core/client/app/templates/setup/three.hbs

```diff
@@ -8,8 +8,7 @@
 
     <form class="gh-flow-invite">
         <label>Enter one email address per line, we’ll handle the rest! <i class="icon-mail"></i></label>
-        {{textarea class="gh-input" name="users" placeholder="john@example.com
-sally.sanders@example.com" value=users}}
+        {{textarea class="gh-input" name="users" placeholder="john@example.com sally.sanders@example.com" value=users}}
```

Comment:
This placeholder is misleading; names are meant to be separated by newlines, not spaces. Better to just remove the `sally.sanders@example.com` part entirely